--- opt/quadpbx/modules/framework/current/amp_conf/htdocs/admin/bootstrap.php	2025-09-14 16:29:51.398016201 +1000
+++ opt/quadpbx/modules/framework/current/amp_conf/htdocs/admin/nowhoops-bootstrap.php	2025-09-14 16:30:30.702409885 +1000
@@ -38,6 +38,9 @@
  * $bootstrap_settings['astman_connected'] = false/false;
  * $bootstrap_settings['function_modules_included'] = true/false true if one or more were included, false if all were skipped;
  */
+
+$bootstrap_settings['freepbx_error_handler'] = false;
+
 if (isset($bootstrap_settings['returnimmediately'])) {
 	return;
 }
